webapp/src/container.tsx

```diff
@@ -613,7 +612,7 @@ export class MainMenu extends data.Component<ISettingsProps, {}> {
                 </a> : undefined}
             </div>
         </div>;
-        /* tslint:enable:react-a11y-anchors */
+        /* eslint-enable jsx-a11y/anchor-is-valid */
```

Comment:
should there be a matching "disable" for this "enable" ?

Reply:
Hmm, it's not complaining so maybe we don't need the enable either. The rules aren't exactly 1-1